index.html

```diff
@@ -138,22 +138,23 @@ <h2>
                         <div class="mathWorkArea">
                             <div class="row">
                                 <div class="col-lg-12">
-                                    <div id="workArea" class="mathEditor"
-                                        style="border: 1px solid #CCC; border-radius: 5px; background-color: #FFF; min-height: 100px; display: inline-block; width: inherit;"
+                                    <div id="workArea" class="mathEditor d-flex flex-nowrap justify-content-between"
+                                        style="border: 1px solid #CCC; border-radius: 5px; background-color: #FFF; min-height: 100px; width: inherit;"
                                         data-step="1"
                                         data-position="top"
                                         data-intro="Type or edit the equation using your keyboard and the math keys below. Try using the cross out, replace, and calc buttons to help show your work.">
                                         <h3 class="sr-only">Math editor</h3>
                                         <div aria-label="edit equation" id='mathEditorActive' role="application" tabindex="0"
-                                            style="float: left; min-width: 400px; margin: 15px 0px 15px 15px;"
+                                            class="order-1"
+                                            style="min-width: 400px; margin: 15px 0px 15px 15px;"
                                             onfocus="TheActiveMathField.focus();"></div>
-                                        <div style="float: right;">
+                                        <div class="order-2">
                                             <h3 id="mathAnnotationHeader" class="mathAnnotation sr-only"
-                                                style="float: right; border-left: 1px solid #CCC;" tabindex="-1">Describe your
+                                                style="border-left: 1px solid #CCC;" tabindex="-1">Describe your
                                                 work</h3>
                                             <textarea class="form-control" id="mathAnnotation"
                                                     placeholder="Use the microphone button or type to explain your work (required)"
-                                                    style="width: 280px; height: 98px; border-left: 1px solid #CCC;"
+                                                    style="width: 280px; height: 98px; margin-top: 1px; margin-bottom: -6px; border-left: 1px solid #CCC;"
```

Comment:
I'd recommend to move it to style css -> #mathAnnotation

Reply:
ok